solr/core/src/test/org/apache/solr/search/TestStressVersions.java

```diff
@@ -116,7 +112,6 @@ public void run() {
                   }
 
                   int id = rand.nextInt(ndocs);
```

Comment:
```suggestion
                  int id = rand.nextInt(ndocs);
                  // Object sync = syncArr[id];
```

because there's some commented out `synchronized (sync)` code below